tests/input_files/contact3D_exodus_in.4C.yaml

```diff
@@ -0,0 +1,82 @@
+PROBLEM SIZE:
+  ELEMENTS: 2000
+  NODES: 2662
+  MATERIALS: 2
+  NUMDF: 6
+PROBLEM TYPE:
+  PROBLEMTYPE: "Structure"
+
+STRUCTURE GEOMETRY:
+  FILE: "contact3D_exodus_in.exo"
+  ELEMENT_BLOCKS:
+    - ID: 1
+      ELEMENT_NAME: SOLID
+      ELEMENT_DATA: "MAT 1 KINEM nonlinear"
+    - ID: 2
+      ELEMENT_NAME: SOLID
+      ELEMENT_DATA: "MAT 2 KINEM nonlinear"
```

Comment:
2. This new section tells us which file to read and which elements to assign. This should be very familiar to pre_exodus users, since it is exactly what you entered in the `bc` file before. Note that the `ELEMENT_DATA` should of course become a clean yaml structure like

```
ELEMENT_DATA:
  MAT: 2
  KINEM: nonlinear
```

but this can also be tackled later. The inline dat-style string is closer to the old code, so I kept this for now.